src/main/tut/index.html

```diff
@@ -445,6 +488,7 @@
 --
 
 We have `Either` to represent these cases:
+<!-- COMMENT mention biasing? -->
```

Comment:
Whilst it may be a rabbit hole you didn't want to get into, but (not sure if you're aware) as `Either` is not actually a monad - only its projects are - its unbiased, so there's no map functions or useful utilities. `cats.Xor` or `scalaz.\/` are both right-biased disjunctions that allow for more generally useful operations.